data/journal_check/bug_refs.json

```diff
@@ -515,7 +515,7 @@
         "type": "bug"
     },
     "bsc#1215378": {
-        "description": "(libvirtd|virtqemud).*: error from service: GDBus.Error:org.freedesktop.login1.OperationInProgress: The operation inhibition has been requested for is already running",
+        "description": "(libvirtd|virtqemud).*: error from service: GDBus.Error:org.freedesktop.(login1.OperationInProgress: The operation inhibition has been requested for is already running|systemd1.ShuttingDown: Refusing activation, D-Bus is shutting down)",
```

Comment:
why that additional `(` ?

Reply:
https://openqa.suse.de/tests/14167463#step/journal_check/22

`Apr 29 15:53:25.414099 localhost.localdomain virtqemud[979]: error from service: GDBus.Error:org.freedesktop.systemd1.ShuttingDown: Refusing activation, D-Bus is shutting down.
`

Reply:
No, what I mean is you replaced
`GDBus.Error:org.freedesktop.login1.OperationInProgress`
by
`GDBus.Error:org.freedesktop.(login1.OperationInProgress`
in the original error message.

Reply:
ok, never mind, I dind't see the closing `)` at the end...